sdk/storage/azblob/internal/testcommon/clients_auth.go

```diff
@@ -128,48 +128,50 @@ func GetServiceClient(t *testing.T, accountType TestAccountType, options *servic
 	return serviceClient, err
 }
 
-func GetAccountInfo(accountType TestAccountType) (string, string) {
+func GetGenericAccountInfo(accountType TestAccountType) (string, string) {
+	if recording.GetRecordMode() == recording.PlaybackMode {
+		return FakeStorageAccount, "ZmFrZQ=="
```

Comment:
Is this just a fake key or this means something?

Reply:
just a fake key for playback testing I think. I was centralizing some of this logic to add a fake key